PWGCF/MultiparticleCorrelations/Tasks/CMakeLists.txt

```diff
@@ -20,6 +20,6 @@ o2physics_add_dpl_workflow(multiparticle-correlations-ar
                     COMPONENT_NAME Analysis)
 
 o2physics_add_dpl_workflow(threeparticle-correlations
```

Comment:
Is it feasible to change it to `three-particle-correlations` in the next iteration?